### DISCUSSION TITLE
PFX

### DISCUSSION THREAD

**Discussion by Faldebbagh** [Digitaler-Impfnachweis/certification-apis#129] [Q&A] [answered]
woher bekomme ich die PFX Datei oder wie soll genierte werden

**Comment by pbroska** [comment, marked as answer]
PFX bzw. Client Zertifikate benötigen nur die Impfzentren und ähnliche Institutionen ohne SMC-B. Das Token Verfahren mit der SMC-B übernimmt den Part der Authentifizierung.
